pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -30,12 +32,8 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
     TEMPORARY_DOMAINS.find((domain) => {
       return domain.title === portfolio.domain;
     });
-  const component =
-    portfolio &&
-    TEMPORARY_COMPONENTS.find((component) => {
-      return component.id === "aa78c93b-d57d-4d9a-b255-e8cfe26bde48";
-    });
-  const badgeNumber = 1;
+  const components = TEMPORARY_COMPONENTS;
+  const badgeNumber = components.results.length;
```

Comment:
No need to assign this, I think, we could just do:

```
const badgeLabel = badgeNumber < 100 ? _.toString(components.results.length) : "99+";
```

Reply:
Edit: we could use the assignment later in the file, I think.

However, `badgeNumber` doesn't say much about _what_ it actually represents. Maybe `componentCount` instead?